modules/utils/docker/entrypoint.sh

```diff
@@ -30,6 +28,7 @@ configure()
       mesh_if=$MESH_VIF
       meshVersion=$MSVERSION
       ML=$ML
+      bridge=$BRIDGE
```

Comment:
@govindsi  @martin-tii  I've changed it to get bridge mode from /opt/mesh.conf now